piDIOComm.c

```diff
@@ -139,6 +139,8 @@ INT32U piDIOComm_Init(INT8U i8uDevice_p)
 
 			ret = pibridge_req_io(addr, IOP_TYP1_CMD_CFG, snd_buf,
 					      snd_len, NULL, 0);
+			if (ret > 0) /* success */
+				ret = 0;
```

Comment:
Change not necessary, you're expecting an empty payload as response, so anything greater than 0 is an error.

Reply:
Indeed